src/components/Form/index.js

```diff
@@ -1,29 +1,14 @@
 import SubmitButton from "./SubmitButton";
-import InputBorrowAmounts from "./InputBorrowAmounts";
-import InputAnnualSalary from "./InputAnnualSalary";
-import InputMonthlyPercentage from "./InputMonthlyPercentage";
-import {useEffect, useState} from "react";
-
-//Question:
-// should I separate the calculations into a separate component?
-// How many components can i break this down into?
-//  If more components then how to lift/prop the inputted values?
-// How can I make it drier?
-
+import MainInputs from "./MainInputs";
 import Calculator from "./Calculator";
-import {click} from "@testing-library/user-event/dist/click";
+import {useState} from "react";
 
 
 const Form = () => {
     const [click, setClick] = useState(0)
```

Comment:
The 'react' way to do this would be to have a onSubmit event listener on the form tag `<form onSubmit={something} />` that triggers some function to run, which in turn does all the maths and updates the state.

However, I do kinda like this trick, the alternative might require moving some bits of state around, or possibly moving some components.